web/geo/generateWorld.ts

```diff
@@ -29,7 +29,7 @@ const EXCHANGE_OUT_PATH = path.resolve(
 
 const fc: WorldFeatureCollection = getJSON(config.WORLD_PATH);
 const zoneConfig = mergeZones();
-const aggregates = generateAggregates(fc, zoneConfig);
+const aggregates = generateAggregates(fc, zoneConfig.zonesConfig);
```

Comment:
In all the places where this is changed, could we rename it from `zoneConfig.zonesConfig` to something less confusing? 😄 
Maybe `zones.config` or just `zones.zonesConfig`?

Reply:
I was thinking about this after I created the PR. I think maybe config.zones make sense then we also have config.contributors.

In the future we could also add the exchanges to the same object but I haven't explored that.

Reply:
Oh, guess we could rename the other `config` constant otherwise?